src/views/v2/month/day-marker.php

```diff
@@ -14,5 +14,5 @@
  */
 ?>
 <div class="tribe-events-c-day-marker">
-	<time class="tribe-events-c-day-marker__date tribe-common-b1 tribe-common-b1--bold" datetime="2019-05-12">May 12</time>
+	<time class="tribe-events-c-day-marker__date tribe-common-h6 tribe-common-h6--alt" datetime="2019-05-12">May 12</time>
```

Comment:
isn't this h7 on mobile and h6 on desktop?

Reply:
Yep, typo, thanks! good catch.